### ISSUE TITLE
Upgrade fs2-reactive-streams 0.6.0

### ISSUE THREAD

**Issue by juanpedromoreno** [higherkindness/mu#303]
fs2-reactive-streams [0.6.0](https://github.com/zainab-ali/fs2-reactive-streams/releases/tag/v0.6.0) is out. We could upgrade adapting our current codebase to the changes that the latest versions of `fs2` and `cats-effect` are bringing.

**Comment by pepegar** [comment]
code to reproduce the bincompat error:

```
import monix.eval.Task
implicit val scheduler = monix.execution.Scheduler.Implicits.global
import cats.effect.IO
implicit val timer = scheduler.timer[IO]
Task(3).toIO

java.lang.NoSuchMethodError: cats.effect.IO$.ioConcurrentEffect()Lcats/effect/ConcurrentEffect;
  at monix.eval.Task.toIO(Task.scala:1622)
  ... 36 elided
```

**Comment by juanpedromoreno** [comment]
Let's wait for https://github.com/monix/monix/pull/681

**Comment by juanpedromoreno** [comment]
Fixed by #472
